### PR TITLE
Complete guide to System Design 2023 By Educative

### PR DESCRIPTION
**Pull request by Syed007Hassan**
- I found this blog very useful as it includes many different topics from System Design. 
- Also, the blog itself has combined many resources. 
- https://www.educative.io/blog/complete-guide-to-system-design

### CODE REVIEW

**Comment by GauravDhak** [comment]
Wonderful Blog @Syed007Hassan Thanks 👍

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ This repository contains System Design resources which are useful to learn conce
 - [API Gateway](https://www.nginx.com/learn/api-gateway/)
 - [Distributed Locking](https://martin.kleppmann.com/2016/02/08/how-to-do-distributed-locking.html)
 - [Checksum](https://www.lifewire.com/what-does-checksum-mean-2625825)
+- [The Complete Guide To System Design in 2023](https://www.educative.io/blog/complete-guide-to-system-design)
```
Comment (ashishps1) [inline README.md:39]: This one contains multiple topics in a single blog and it shouldn't be under this section. Maybe we need to create a different section for these types of blogs.

Comment (Syed007Hassan) [reply README.md:39]: It makes sense to have another section for such blogs.

Comment (GauravDhak) [reply README.md:39]: Yaa, Definitely Create another updated Redme.md file

**Comment by GauravDhak** [comment]
I found this blog very useful topics in System Design.
Also, the blog itself has combined many resources.
#https://tikv.org/deep-dive/key-value-engine/b-tree-vs-lsm/

**Pull request closed without merging**